src/checkstyle/intellij-code-style-scheme.xml

```diff
@@ -0,0 +1,603 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    This file can be imported into IntelliJ to configure editor settings
+    for Google Style conventions.`
+-->
+<code_scheme name="GoogleStyle">
```

Comment:
What's the original source for this file? Maybe just point people to the source in the readme?

Reply:
Done - I realized this is linked in the contributing.md so we are good. Removed.